docs/datatypes/core/zio/zio.md

```diff
@@ -602,6 +602,40 @@ Also, this operation may throw unexpected errors like `OutOfMemoryError`, `Stack
 
 Therefore, it is quite common to import a code that may throw exceptions, whether that uses expected errors for error handling or can fail for a wide variety of unexpected errors like disk unavailable, service unavailable, and so on. Generally, importing these operations end up represented as a `Task` (`ZIO[Any, Throwable, A]`). So in order to make recoverable errors typed, we use the `ZIO#refineOrDie` method.
 
+## Sandboxing
+
+To expose full cause of a failure we can use `ZIO#sandbox` operator:
+
+```scala mdoc:silent
+import zio._
+def validateCause(age: Int) =
+  validate(age) // ZIO[Any, AgeValidationException, Int]
+    .sandbox    // ZIO[Any, Cause[AgeValidationException], Int]
+```
+
+Now we can see all the failures that occurred, as a type of `Case[E]` at the error channel of the `ZIO` data type. So we can use normal error-handling operators:
```

Comment:
Did you mean `Cause[E]` instead of `Case[E]` here?

Reply:
Thank you for reporting. fixed!